client/keys/import.go

```diff
@@ -38,3 +38,19 @@ func ImportKeyCommand() *cobra.Command {
 		},
 	}
 }
+
+func ImportKeyHexCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "import-hex <name> <hex> <key-type>",
```

Comment:
Maybe it is a better UX if the 3rd argument is optional and have a default algo string?

Reply:
yes, completely agree!! nice catch!

Reply:
key-type is now a flag with secp256k1 as default value